jsk_spot_robot/jsk_spot_startup/launch/include/driver.launch

```diff
@@ -0,0 +1,170 @@
+<launch>
+  <arg name="credential_config" default="/var/lib/robot/credentials/spot_credential.yaml" />
+  <arg name="launch_robot_state_publisher" default="true" />
+  <arg name="launch_sound_play" default="true" />
+
+  <arg name="use_voice_text" default="false" doc="voice_text needs license" />
+  <arg name="use_app_manager" default="true" />
+
+
+  <group if="$(arg launch_robot_state_publisher)">
+    <include file="$(find spot_description)/launch/description.launch" />
+  </group>
+
+  <node
+      pkg="spot_driver"
+      type="spot_ros"
+      name="spot_ros"
+      ns="spot"
+      output="screen"
+      respawn="true"
+      >
+    <rosparam file="$(find spot_driver)/config/spot_ros.yaml" command="load" />
+    <rosparam file="$(arg credential_config)" command="load" />
+    <remap from="joint_states" to="/joint_states"/>
+    <remap from="tf" to="/tf"/>
+  </node>
+
+  <node name="cmd_vel_smoother" pkg="cmd_vel_smoother" type="cmd_vel_smoother">
+    <remap from="input" to="spot/cmd_vel_unsmoothed" />
+    <remap from="output" to="spot/cmd_vel" />
+
+    <param name="desired_rate" value="10.0" />
+    <param name="x_acc_lim" value="1.0" />
+    <param name="y_acc_lim" value="0.5" />
+    <param name="yaw_acc_lim" value="0.1" />
+    <param name="decel_factor" value="0.8" />
+    <param name="max_count" value="5" />
+  </node>
+
+  <node
+      pkg="twist_mux"
+      type="twist_mux"
+      name="twist_mux"
+      >
+    <rosparam command="load" file="$(find jsk_spot_startup)/config/twist_mux.yaml" />
+    <remap from="cmd_vel_out" to="spot/cmd_vel_unsmoothed"/>
+  </node>
+
+  <!-- teleop -->
+  <node pkg="teleop_twist_joy" type="teleop_node" name="teleop_twist_joy_head" >
+    <remap from="joy" to="joy_head/joy_complemented" />  <!-- apps/head_lead_demo.xml -->
+    <remap from="cmd_vel" to="joy_head/cmd_vel" />
+    <rosparam command="load" file="$(find jsk_robot_startup)/config/head_teleop_twist_joy.yaml" />
+  </node>
+
+
+  <!-- joystick teleop nodes -->
+  <include file="$(find jsk_robot_startup)/launch/quadruped_joystick_teleop.launch" >
+    <arg name="pad_type" value="dualsense" />
+    <arg name="joy_dev" value="/dev/input/js0" />
```

Comment:
it is better to write udev file as below and use `/dev/dualsense` for device path.
otherwise, the first joy device will be used for teleoperation.
Belka has this internal change to use different joystick and dualsense

```
KERNEL=="js*", ATTRS{name}=="Wireless Controller", SYMLINK+="dualsense"
```